release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2866,3 +2866,29 @@ link:https://access.redhat.com/solutions/6956858[{product-title} 4.10.13 contain
 ==== Updating
 
 To upgrade an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-10-14"]
+=== RHBA-2022: - {product-title} 4.10.14 bug fix update
+
+Issued: 2022-05-18
+
+{product-title} release 4.10.14 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:2178[RHBA-2022:2178] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:2177[RHBA-2022:2177] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6958424[{product-title} 4.10.14 container image list]
+
+[id="ocp-4-10-14-features"]
+==== Features
+
+[id="ocp-4-9-7-update-mode-update-cluster"]
+===== Update the control plane independently of other worker nodes
+
+With this update, you are now able to do a partial cluster update within the *Update Cluster* modal. You are able to update the worker or custom pool Nodes to accommodate the time it takes for maintenance. You can also pause and un-pause within each pool's progress bar. An alert will display at the top of the *Cluster Settings* page if one or more worker or custom pools are paused. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076777[*BZ#2076777*])
+For more information, see ...
```

Comment:
TBD looks like

Reply:
Yes may be one or 2 xrefs. Not sure yet.

Reply:
LGTM